src/Admin/AdminInterface.php

```diff
@@ -700,12 +574,12 @@ public function showMosaicButton(bool $isShown): void;
      */
     public function isDefaultFilter(string $name): bool;
 
-    public function configureActionButtons(array $buttonList, string $action, ?object $object = null): array;
+    public function configureActionButtons(array $buttonList, string $action, object $object = null): array;
 
     /**
      * Check object existence and access, without throwing Exception.
      */
-    public function canAccessObject(string $action, object $object): bool;
+    public function canAccessObject(string $action, object $object = null): bool;
```

Comment:
IMO, `?object = null` is redundant: while `?` is intended to expose that the argument is required and can accept a `null` value, ` = null` is allowing the `null` value and providing a default value (making the argument optional).

Reply:
I think we should make a decision and add the `nullable_type_declaration_for_default_null_value` option for `phpcs-fixer`.

Personally, I prefer to use `?object = null`. This way I just need to look at the signature to know every allowed value. And if someone change the default value, it doesn't change the allowed values.

Reply:
@phansys This will be certainly be merged https://github.com/sonata-project/dev-kit/pull/661 so I think you can already start to write `?object = null`

Reply:
I've applied the rule under `src/Admin/`, which is the directory where the changes of this PR are focused.